lib/index.js

```diff
@@ -74,12 +78,6 @@ Appboy.prototype.loaded = function() {
  */
 
 Appboy.prototype.identify = function(identify) {
-  var userId = identify.userId();
-  // Appboy doesn't want you to call changeUser unless a userId is present because they do automatic anonymous user tracking on their end 
-  // https://www.appboy.com/documentation/Web/#setting-user-ids
-  if (!userId) return;
-  window.appboy.changeUser(userId);
```

Comment:
yup, they said this implementation was fine since we are calling it in initialize